MATLAB/environmental_models/env_atmospheric_drag.m

```diff
@@ -0,0 +1,89 @@
+function F_envdrag = env_atmospheric_drag(time,r,v)
+%input: mission time in sec
+% r position in ECI frame in m
```

Comment:
Input position in the ECEF frame, instead of ECI frame. Then you can use, `ecef2lla` instead of `eci2lla` which is much faster and you won't need to deal with all of the UTC leap second, datatime stuff.

Reply:
Actually, maybe input both, because you need position in ECI for `v_rel = v - cross(const.earth_rate_ecef,r);` to calculate velocity relative to the rotating atmosphere?

Reply:
updated